New feat: load database url parameter from dotenv file

autokludge opened the pull request:
Additional dependencies: 
- dotenvy
- clap --feature env

Load envvars from the .env file with in the current dir and override existing envvars if they exist (`dotenvy`).
Use `DATABASE_URL` envvar if the `--url` argument isn't passed to the subcommand

autokludge committed:
load database url parameter from dotenv file

mattyoung101 commented:
Thanks for the PR, this looks good! Could you also please add some documentation to the `README.md` documenting the presence of the new `DATABASE_URL` env var?

autokludge committed:
add DATABASE_URL envvar info to README

edtear will now use DATABASE_URL if it is set in .env
README.md updated to document this new feature
